src/ngraph/op/scatter_nd_update.cpp

```diff
@@ -0,0 +1,98 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/scatter_nd_update.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/shape.hpp"
+using namespace std;
+using namespace ngraph;
+
+static int INPUTS = 0;
+static int INDICES = 1;
+static int UPDATES = 2;
+
+constexpr NodeTypeInfo op::v3::ScatterNDUpdate::type_info;
+
+shared_ptr<Node> op::v3::ScatterNDUpdate::clone_with_new_inputs(const OutputVector& new_args) const
```

Comment:
Visitor is missing

Reply:
@diyessi @lazarevevgeny , i have addressed the feedback, can you pls approve if it looks good.